x-pack/plugins/profiling/public/views/add_data_view/index.tsx

```diff
@@ -67,192 +82,197 @@ export function AddDataView() {
+              defaultMessage: 'Run the Universal Profiling container:',
+            }),
+            content: (
+              <EuiCodeBlock paddingSize="s" isCopyable>
+                {`docker run --name pf-host-agent --privileged --pid=host -v /etc/machine-id:/etc/machine-id:ro \\
 -v /var/run/docker.sock:/var/run/docker.sock -v /sys/kernel/debug:/sys/kernel/debug:ro \\
 docker.elastic.co/observability/profiling-agent:${stackVersion} /root/pf-host-agent \\
 -project-id=1 -secret-token=${secretToken} \\
 -collection-agent=${collectionAgentHost}`}
-            </EuiCodeBlock>
-          ),
-        },
-      ],
-    },
-    {
-      key: AddDataTabs.Binary,
-      title: i18n.translate('xpack.profiling.tabs.binaryTitle', {
-        defaultMessage: 'Binary',
-      }),
-      steps: [
-        {
-          title: i18n.translate('xpack.profiling.tabs.binaryDownloadStep', {
-            defaultMessage: 'Download the binary for the right architecture:',
-          }),
-          content: (
-            <EuiText>
-              <b>For x86_64:</b>
-              <EuiSpacer size="s" />
-              <EuiCodeBlock paddingSize="s" isCopyable>
-                {`wget -O pf-host-agent.tgz "https://artifacts.elastic.co/downloads/prodfiler/pf-host-agent-${stackVersion}-linux-x86_64.tar.gz" && tar xzf pf-host-agent.tgz`}
               </EuiCodeBlock>
-              <EuiSpacer size="m" />
-              <b>For ARM64:</b>
-              <EuiSpacer size="s" />
+            ),
+          },
+        ],
+      },
+      {
+        key: AddDataTabs.Binary,
+        title: i18n.translate('xpack.profiling.tabs.binaryTitle', {
+          defaultMessage: 'Binary',
+        }),
+        // Create a dedicated sub tab for each architecture that we support
+        subTabs: supportedCPUArchitectures.map((arch) => {
+          return {
+            key: arch,
+            title: `Linux ${arch}`,
+            steps: [
+              {
+                title: i18n.translate('xpack.profiling.tabs.binaryDownloadStep', {
+                  defaultMessage: 'Download the binary:',
+                }),
+                content: (
+                  <EuiCodeBlock paddingSize="s" isCopyable>
+                    {`wget -O pf-host-agent.tgz "https://artifacts.elastic.co/downloads/prodfiler/pf-host-agent-${stackVersion}-linux-${arch}.tar.gz" && tar xzf pf-host-agent.tgz`}
+                  </EuiCodeBlock>
+                ),
+              },
+              {
+                title: i18n.translate('xpack.profiling.tabs.binaryGrantPermissionStep', {
+                  defaultMessage: 'Grant executable permissions:',
+                }),
+                content: (
+                  <EuiCodeBlock paddingSize="s" isCopyable>
+                    {i18n.translate('xpack.profiling.tabs.binaryEnsurePermissions', {
+                      defaultMessage: `chmod +x pf-host-agent-${stackVersion}-linux-${arch}/pf-host-agent`,
+                    })}
```

Comment:
Ah, I got what's the problem. You should not use template literals. You must not use the Backtick and `${arch}` to define a variable. You should instead do:
```suggestion
                    {i18n.translate('xpack.profiling.tabs.binaryEnsurePermissions', {
                      defaultMessage: 'chmod +x pf-host-agent-{stackVersion}-linux-{arch}/pf-host-agent',
                    }, values: {
                    stackVersion, 
                    arch
                    })}
```

This should fix the problem.